### DISCUSSION TITLE
Documentation styling question: How to represent OS-specific items

### DISCUSSION THREAD

**Discussion by ClaytonCurtis1947** [freeplane/freeplane#1228] [Help] [answered]
I am trying to contribute to the community by helping with documentation, and am coming across things where I need guidance.
Apologies in advance for a long post!

A. Is there some place that describes conventions for documentation styling that would address my next question?

B. At 'https://docs.freeplane.org/user-documentation/templates.html' there is a space for Windows paths to system and user-defined templates that currently reads "TODO', and I'm trying to fill that gap.

The path descriptions for Linux are pretty straightforward:

-----
	**System-wide** templates (when Freeplane is installed via the package manager of your distribution) are located in:
	
	`/usr/share/freeplane/resources/templates`
	
	can be found in:
	
	`~/.config/freeplane/VERSION/templates`
-----

For Windows, example equivalents in concrete form are:
	`\Program Files\Freeplane\resources\templates'
	`\Users\curti\AppData\Roaming\Freeplane\1.11.x\templates'     <<<<<  I'm the 'curti'

Q1: Am I correct that the representation of the install directory under Windows as shown is acceptable?

Q2: How does one represent generalization or abstraction of a path element. For example, is '/VERSION/' in the Linux case actual text that would appear or an abstraction of the concept of version?

For Windows, there is no abstraction for version. There is no abstraction for user other than plugging in an environment variable such as 'USERNAME', which would be represented in a path as C:\Users\%USERNAME%\.  So should the path to **User-defined** templates be represented as
    '\Users\%USERNAME%\AppData\Roaming\Freeplane\1.11.x\templates' -- which still uses an explicit specification of version and leaves open the question of representing an abstraction. (ref. the mention of VERSION above.)  And raises the question of the appropriateness of using standard (system-defined) environment variables.

Thanks!

**Comment by dpolivaev** [comment, marked as answer]
Your questions make sense. However, consistency requires strong governance and at least in my eyes, the documentation project does not have strong leadership or governing structures. So I think that the pragmatic decision is any decision you take, unless you want to suggest general principles and spend your energy to install them.

Let me tell you my perspective about the goals of this freeplane documentation project: here with GitHub, we have an easy enough documentation tool where potentially all people can contribute in order to share their knowledge and make use of Freeplane easier for other people. Then some lack of consistency does not seem to be critical and all improvements are possible and welcome.
